apps/builder/app/builder/features/props-panel/control.tsx

```diff
@@ -356,17 +323,26 @@ export function Control({
             })
           }
           options={options ?? DEFAULT_OPTIONS}
-          type={type}
+          type={control}
         />
       );
     }
 
-    assertUnreachable(type, `Unknown control type ${type}`);
+    if (
+      control === "object" ||
+      control === "date" ||
+      control === "range" ||
+      control === "multi-select"
```

Comment:
multi-select can be rendered as check, seems same data, same result, just a different way of selecting multiple items